com.wire.WireDesktop.json

```diff
@@ -0,0 +1,103 @@
+{
+  "app-id": "com.wire.WireDesktop",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "command": "/app/Wire/wire-desktop",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "separate-locales": false,
+  "sdk": "org.freedesktop.Sdk",
+  "rename-desktop-file": "wire-desktop.desktop",
+  "rename-icon": "wire-desktop",
+  "finish-args": [
+    "--device=all",
+    "--device=dri",
+    "--filesystem=home:ro",
+    "--filesystem=xdg-download",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "modules": [
+    {
+      "name": "libnotify",
+      "cleanup": [
+        "/bin"
+      ],
+      "config-opts": [
+        "--disable-introspection",
+        "--disable-static",
+        "--disable-tests"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "9cb4ce315b2655860c524d46b56010874214ec27e854086c1a1d0260137efc04",
+          "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.7.tar.xz"
+        }
+      ]
+    },
+    {
+      "name": "libsecret",
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/*.la",
+        "/lib/pkgconfig"
+      ],
+      "config-opts": [
+        "--disable-gtk-doc",
+        "--disable-introspection",
+        "--disable-manpages",
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "f4fa198d4e131f94d9ec84cfeaf6c124245e7ff3c0af11c76cc35332bfae5585",
+          "url": "https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.18.6/libsecret-0.18.6.tar.gz"
+        }
+      ]
+    },
+    {
+      "name": "wire",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x wire_*.deb",
+        "tar xf data.tar.xz",
+        "rm -f control.tar.gz data.tar.xz debian-binary wire_*.deb",
+        "cp -r opt/* usr/* /app",
+        "desktop-file-edit --remove-key=Version /app/share/applications/wire-desktop.desktop",
```

Comment:
I created a new issue wireapp/wire-desktop/issues/1768 to fix this one; whenever that makes it into a release I'll delete this line.